### PR TITLE
Pre-Commit test

### PR DESCRIPTION
**Pull request by RitikJaiswal75**
Add a script test to pre-commit so that tests run before committing any changes.

### CODE REVIEW

File: src/constants/commands.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  name: "link",
+  description:
+    "Generate a link with user specific token to link with RDS backend.",
+};
+
+export const APPLY_CHANGE = {
+  name: "change",
+  description:
+    "command to be run after /link to change discord details of user.",
+};
```
Comment (samyakshah3008) [inline src/constants/commands.ts:16]: If this is specifically targets to change the discord details of user, then can we change the const name from "APPLY_CHANGE" to "CHANGE_USERDETAILS" ?

Comment (samyakshah3008) [inline src/constants/commands.ts:16]: If this specifically targets to change the discord details of user, then can we change the const name from "APPLY_CHANGE" to "CHANGE_USERDETAILS" ?

**Comment by sahsisunny** [comment]
Can you tell me please why the same changes in different PR's [In this](https://github.com/Real-Dev-Squad/discord-slash-commands/pull/14)

**Comment by RitikJaiswal75** [comment]
> Can you tell me please why the same changes in different PR's. [In this](https://github.com/Real-Dev-Squad/discord-slash-commands/pull/14)

This is because this pr takes a pull from the previous one. once the previous PR is approved and merged these changes will disappear.
THus making the PR small

**Review by vinayak-trivedi — APPROVED** [review]
LGTM

File: src/constants/commands.ts (hunk 1 above)
Comment (RitikJaiswal75) [reply src/constants/commands.ts]: Naming could be refactored.

Comment (RitikJaiswal75) [reply src/constants/commands.ts]: Also this is not a part of this PR

**Pull request merged**